### ISSUE TITLE
Feature request: quick-connect button

### ISSUE THREAD

**Issue by djechelon** [schwabe/ics-openvpn#1221]
I'd like to request a new feature: the ability to drag to home screen a 1x1 button that automatically connects the chosen VPN profile.

Examples:

Hold-tapping on Gmail shows you the list of your inboxes. There you can access your own inbox directly, and drag it to home screen (as I did it).

I don't remember this feature's name, but it could be useful to connect a VPN with a single touch

**Comment by schwabe** [comment]
This already exists. Long press on the Icon of app in newer android version or add a shortcut in older ones.

**Comment by djechelon** [comment]
Didn't work for me on Android 10.

When VPN is disconnected, I see no buttons on long press.

When VPN is connected, I see it and I can access the VPN main window to disconnect it

**Comment by schwabe** [comment]
on the icon in the launcher. Works fine here with the Google Launcher and Nova.

**Comment by djechelon** [comment]
Here is Pixel 3xl with stock Android 10.0 up to date to July 2020.
No custom launcher

![Screenshot_20200803-111401](https://user-images.githubusercontent.com/3042285/89171001-cc662d80-d580-11ea-9757-c44eda2506d3.png)


Obviously, I don't want to look harsh. I honestly thought this feature was unavailable, and then we discovered that there is something wrong

**Comment by schwabe** [comment]
Pixel 2 XL:
![image](https://user-images.githubusercontent.com/1038539/89202169-d6a11f80-d5b2-11ea-8792-959c7e3a1d68.png)
Not sure why your phone is behaving different.

**Comment by djechelon** [comment]
Got it!!! I think I am talking about a different app. The icons are different!

I have downloaded OpenVPN Connect (`net.openvpn.openvpn`), which sounds official from OpenVPN Inc.

I (wrongly) _assumed_ the app to be open source like the various clients (OpenVPN Inc, as a commercial company, offers both FOSS and premium services), so I looked for an issue tracker and I stumbled upon this app, which is `de.blinkt.openvpn`.

I shall make sure that `net.openvpn.connect` is FOSS, and if that's so to find their own issue tracker.

Sorry for the hassle. Looks like there is a feature that the..... _official_ client doesn't support 😄
